pkg/reconciler/apis/apiexport/apiexport_controller.go

```diff
@@ -202,7 +202,7 @@ type controller struct {
 	commit                     CommitFunc
 }
 
-// enqueueAPIBinding enqueues an APIExport .
+// enqueueAPIExport enqueues an APIExport.
```

Comment:
nit: not including unrelated stuff helps to review :)

Reply:
it was in a separate commit this time :) , not good enough?